src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let sum = days * 40;
+  const firstDiscount = 20;
```

Comment:
Consider using more descriptive constant names, such as `smallDiscount` and `largeDiscount` instead of `firstDiscount` and `secondDiscount`.